source/_integrations/mqtt.markdown

```diff
@@ -119,6 +119,11 @@ MQTT (aka MQ Telemetry Transport) is a machine-to-machine or "Internet of Things
 
 Your first step to get MQTT and Home Assistant working is to choose a broker.
 
+The easiest option is to install the official Mosquitto Broker add-on. You can choose to set up and configure this add-on automatically when you set up the MQTT integration. Home Assistant will automatically generate and assign a safe username and password, and further attention is required. This also works if you have already set up this add-on yourself in advance.
```

Comment:
should this be "and no further attention is required"?

Reply:
Nice catch! I'll change it!